exporters/exporter.go

```diff
@@ -202,6 +202,15 @@ func NewExporter(name, prefix, cloud string, disabledMetrics []string, endpointT
 		log.Infoln("SSL verification disabled on transport")
 		tlsConfig := &tls.Config{InsecureSkipVerify: true}
 		transport = &http.Transport{TLSClientConfig: tlsConfig}
+	} else if config.CACertFile != "" {
```

Comment:
just use a if, else if adds unneeded indenting.

Reply:
That would break functionality for `verify: False` if both verify and `cacert` is set.

Reply:
> That would break functionality for `verify: False` if both verify and `cacert` is set.

Perhaps, its a bit of refactoring needed here.

just use a if, else if adds unneeded indenting.

```suggestion
var transport http.Transport
var tlsConfig tls.Config

if config.CACertFile != "" {
    //your code should return nil, err in case of not found.
    tlsConfig, err := prepareTLSConfig(config.CACertFile)
    if err != nil {
	log.Errorf("Failed to include additional certificates to ca-trust: %v", err)
    } 
}

if config.Verify != nil && !*config.Verify {
    tlsConfig.InsecureSkipVerify = true
}

transport.TLSClientConfig = tlsConfig
```

Reply:
As `prepareTLSConfig` would previously return `*tls.Config` I have changed the function name and return type to easier accomodate the requested change.